src/style.css

```diff
@@ -1,3 +1,117 @@
+html {
+  font-family: "Roboto", "Open Sans", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  font-style: normal;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --white: #fff;
+  --blue: #00acdc;
+}
+
+img {
+  height: 134px;
+  width: 160px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 166px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card-image {
+  display: flex;
+  justify-content: center;
+}
+
+.card-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-accent);
+  margin-top: 40px;
+}
+
+.card-code {
+  color: var(--secondary);
+  margin-top: 4px;
+}
+
+.review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+}
+
+.stars :nth-child(-n + 5) {
```

Comment:
use styles from a `stars` task. Here should be styles for any amount of active stars to make your solution flexible